osvr/RenderKit/RenderManager.h

```diff
@@ -332,14 +332,28 @@ namespace renderkit {
         /// room transform, but will have the room appended.
         class RenderParams {
           public:
-            RenderParams(){};
+            /// @brief Default constructor reads things from a display configuration.
+            /// @param [in] dc OSVRDisplayConfiguration to read values from.  If none
+            ///                is specified, it defaults to the default configuration.
+            RenderParams(OSVRDisplayConfiguration const &dc = OSVRDisplayConfiguration()) {
+              nearClipDistanceMeters = dc.getNearClipMeters();
+              farClipDistanceMeters = dc.getFarClipMeters();
+              IPDMeters = dc.getIPDMeters();
```

Comment:
Yes, it pulls them from the default display config rather than assigning its own defaults in this class the way it used it.  It also enables the client to read the display configuration from the server and pass it in or to go ahead and override them with whatever it chooses the way it used to when the defaults were defined in this file.  Figured it was better to have one place where these were defined rather than two that could (and in fact did) have different values.